appdev/guides/pushnotifications.rst

```diff
@@ -0,0 +1,270 @@
+Push notifications
+==================
+
+Let's assume that you have an app created with Clickable and published on the OpenStore. But now you want to be able to send Push Notifications to your users. First of all, you need to understand how this is working:
+
+.. image:: ../../_static/images/appdev/guides/pushnotifications/Diagram.png
+
+1. Every app which uses push notifications has got a unique token. This token identifies the user, the device and the app itself. The token is generated by the UBports Push Service.
+2. You will need the token to send a push notification. So the app sends its token to the app developer's server.
+3. With the token you can send a HTTP request to the UBports Push Server which will forward the notification the user's device.
+
+Let's practice this step-by-step.
+
+.. note::
+    In the following example we will not implement a server. Also the communication between your app and your server is up to you. Please inform the user about the communication with your server by providing a privacy policy!
+
+Make the app ready for push notifications
+-----------------------------------------
+
+Implementing the PushClient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Implementing the pushclient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Repeated header

Reply:
Don't know how this could happen... maybe a wrong copy&paste action. Sorry... this is fixed now